builders/build/customize/fixing-dependencies-conflicts.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Fixing Dependencies Conflicts
+description: Learn how to manage and resolve double reference issues of dependencies when customizing your Tanssi Appchain template with a Rust-based tool called Diener.
+---
+
+# Fixing Dependencies Conflicts {: #fixing-dependencies-conflicts }
```

Comment:
"Adding a third-party module"? (Or "Adding external dependencies"?)
Having articles about adding "Built-in", "Custom-made", and "Third-Party" feels more continuous

Reply:
We could probably expand a bit this content adding an example step by step, like we have in the other articles, 
 describing how to add a third party dependency. wdyt?